app/assets/stylesheets/application.css

```diff
@@ -1,7 +0,0 @@
-/*
- *= require normalize-rails
- *= require bootstrap-sass/assets/stylesheets/_bootstrap
- *= require leaflet/dist/leaflet
- *= require_self
- *= require_tree .
```

Comment:
I'll have to check how new Rails generate CSS  application file by default, but it was the killer feature to load a whole folder of CSS files. :cry: